docs/resources/metric_alert.md

```diff
@@ -35,6 +35,18 @@ resource "sentry_metric_alert" "main" {
     threshold_type  = 0
   }
 
+   trigger {
+    action {
+      type              = "slack"
+      target_type       = "specific"
+      target_identifier = "#slack-channel"
+      integration_id    = 99999
```

Comment:
Perhaps you could create a comment here / example how you might use this with the `organisation_integration`?